### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by asanchezyali**

### CODE REVIEW

**Review by juanArias8 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -307,38 +258,29 @@ docker compose build api
...
-### Adding a new dependency to the frontend
+## Adding a new dependency to the frontend
 
 ```shell
 # Add a new dependency to the npm package.json file
-docker-compose run --rm client yarn install <dependency>
+yarn install <dependency>
 
 # Update the docker image
 docker-compose build client
```
Comment (curibe) [inline README.md:286]: @asanchezyali please remove the dash from all parts where you find it, in docker compose:   docker-compose -> docker compose

File: README.md [hunk 2]
```diff
@@ -307,38 +258,29 @@ docker compose build api
...
-   ```
+1. Add the new dependency directly to the respective `requirements.txt` file.
+2. Update the docker image: `docker compose build api`.
+3. Run the image: `docker compose up`.
 
-**Note:** This project uses requirements.lint.txt only for caching in ci workflow linting job.
+**Note:** This project uses `requirements.lint.txt`` only for caching in CI workflow linting job.
 
-### Adding a new dependency to the frontend
+## Adding a new dependency to the frontend
```
Comment (curibe) [inline README.md:279]: Don't remove '#' symbol. This is a heading3, not a heading2

File: README.md [hunk 3]
```diff
@@ -44,256 +44,207 @@ using image editing and stable diffusion models.**
...
-
-# Migrate / Update the head
-docker-compose run --rm datalib alembic upgrade head
-```
-
-### PG admin
-
-PGadmin is available in: localhost:8002
+* To create a migration: `docker-compose run --rm datalib alembic revision --autogenerate -m "Initial migration"`.
+* To migrate or update to the head: `docker-compose run --rm datalib alembic upgrade head`.
```
Comment (curibe) [inline README.md:232]: @asanchezyali please change from inline-code to block-code(with three quote marks). It allows one to copy the command more easily, especially for long commands. Please do the same in other parts of the REAMDE where it is required

**Review by curibe — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (curibe) [inline README.md:286]: @asanchezyali please remove the dash from all parts where you find it, in docker compose: docker-compose -> docker compose

**Review by curibe — CHANGES_REQUESTED** [review]

**Review by curibe — APPROVED** [review]

**Pull request merged**